docs/functionaldatatypes/index.md

```diff
@@ -0,0 +1,13 @@
+---
+id: index
+title: "Summary"
+---
+
+ZIO Prelude includes several data types to help us model our domains more accurately and solve common problems.
+
+- **[Equivalence](equivalence.md)** - A description of an equivalence relationship between two data types.
+- **[NonEmptyList](nonemptylist.md)** - A list that is guaranteed to be non-empty to more accurately model situations where we know a collection has at least one element.
+- **[These](these.md)** - A data type that may either be a `Left` with an `A`, a `Right` with a `B`, or a `Both` with an `A` and a `B`, useful for modeling problems such as merging streams of data.
```

Comment:
good statement for `These` with good example.

Reply:
No `NonEmptySet` yet I guess.

Reply:
There actually is but it is a little more of a wrapper for a normal set than a full blown data type.